static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -1203,6 +1204,10 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
     if (alertType === 'custom_metrics') {
       formattedAggregate = formatMRIField(aggregate);
     }
+    const func = parseFunction(aggregate);
+    if (func && alertType === 'eap_metrics') {
```

Comment:
Is there an enum for this? A quick search says no, maybe we should make one in the future.

Reply:
Yeah no enums for this at the moment, `AlertType` is just a list of strings right now. We can update in a future change if needed 👍